src/button/ux-button-theme.css

```diff
@@ -76,6 +76,22 @@ styles.button.flat:active {
   background-color: ${background || $swatches.grey.p300};
 }
 
+styles.button.icon {
+  display: inline-flex;
+  align-content: center;
+  justify-content: center;
+
+  width: 40px;
+  height: 40px;
+
+  background-color: transparent;
+  color: ${foreground || $swatches.grey.p700};

```

Comment:
See the usage above. Currently since we do not have our own icons it just centers an icon element the user can include. Down the line, I would prefer to have a usage similar to.

``` html
<ux-button type="icon" icon="icon-here"></ux-button>
```